main.go

```diff
@@ -69,13 +73,30 @@ func runMain() error {
 	log.Printf("Audio files directory is %s", conf.AudioDir)
 
 	for {
-		err = DownloadAndPlaySounds(conf.AudioDir, soundCard)
-		if err != nil {
-			// Wait until tomorrow.
-			log.Printf("Error playing sounds: %v", err)
-			playlist.WaitUntilNextDay()
+
+		soundsDownloaded := false
+		for i := 0; i < maxRetries; i++ {
+			err = DownloadAndPlaySounds(conf.AudioDir, soundCard)
+			if err != nil {
+				log.Println("Error dowloading sounds and schedule:", err)
+				log.Println("Trying again in", int64(retryInterval/time.Second), "seconds.")
```

Comment:
`time.Duration` instances implement `String()` and return a human readable values so you should just be able to do:
```go
log.Printf("Trying again in %s", retryInterval)
```

Reply:
Hmm, this is going to say it will retry on the last failure when it's actually not going to. You probably need to restructure the loop to do this properly.